### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by oskarrough**
Some small tweaks to improve performance.

I know you were loading the script before styles on purpose but it is faster to put it at the end. This way it doesn't block rendering and styles can be loaded in parallel. It's about 2-300ms faster here.

On a 3G device, moving the styles inline would save ~3-500ms as well, but I did not do that because it is annoying to maintain.

### CODE REVIEW

**Comment by 4www** [comment]
Thanks for this PR.

What exactly are you measuring?
- time for the page to load and render https://find.internet4000.com/?
- time for the page to load after a search was made through the browser's URL bar (so the page loads its JS script, that gets the query and from the query where to redirect) — aka time to redirect?

The performance of loading the website is not the most important, as it is not used by the actual users, but by the "possible future users" as their landing page.

Find users are only using it through the URL bar of their web browser.

In any case, performance improvements are much appreciated since a classic user will probably use find through queries in the URL bar thousands time a day.

Cheers!

**Comment by oskarrough** [comment]
When you use Find! through the URL bar, it still needs to load the website, styles and scripts until it can redirect you to the right URL? 

I was using the network tools in the browser to measure the "time to first interactive", throttled to a "slow 3g".  After your comment I tried many other things, including service worker, but I can't seem to make it load noticable faster than what is currently online. Removing styles also didn't seem to make it faster because we are using HTTP2 and the scripts loads async anyway.

webpagetest
https://www.webpagetest.org/result/181101_BJ_3fa65d699111582e8ae37383b5a50c1e/
https://www.webpagetest.org/result/181101_0Y_00e09a759c4408e6a0de23d2cf50c3f4/

gtmetrix
https://gtmetrix.com/reports/find.internet4000.com/ELFI4fd5
https://gtmetrix.com/reports/gifted-bhabha-4374c3.netlify.com/0CyI3fqH

SO, to summarize. This PR can I think be merged as is, just small tweaks anyway.

If we want to really make it fast, we need to eliminate the need of a request to find.internet4000.com — it's not the website itself that is slow, it is the network connecting to the site. Is there an architecture where the main script could be run without a network request first? But that's for another time...

**Comment by 4www** [comment]
deepinscreenshot_select-area_20181105111525

I still don't understand why we'd put styles at first, in our particular situation, where we want the js to be loaded and executed — since 99% of of use case don't need the html page, style, images to be rendered

You mention Time To Interactive, but how is it pertinent in this situation, where nothing is interactive — all our use will come from using !Find from the URL bar.
